holmes/interactive.py

```
@@ -1432,8 +1433,9 @@ def get_bottom_toolbar():
             )
             break
         except Exception as e:
-            logging.error("An error occurred during interactive mode:", exc_info=e)
-            console.print(f"[bold {ERROR_COLOR}]Error: {e}[/bold {ERROR_COLOR}]")
+            logging.debug("Full traceback for interactive mode error:", exc_info=e)
+            console.print(f"\n[bold {ERROR_COLOR}]Error: {e}[/bold {ERROR_COLOR}]")
+            console.print(f"[dim]Tip: Use -v for more details[/dim]")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)